src/shared/dropdown-list-v2/dropdown-list.tsx

```diff
@@ -359,6 +376,7 @@ export const DropdownList = <T, V>({
                 <ListItem
                     aria-selected={selected}
                     aria-multiselectable={multiSelect}
+                    aria-disabled={hasSelectedMax}
```

Comment:
on further thought, for this and line 391, the item should only be disabled if it is `hasSelectedMax` and is NOT selected

because by right the selected items are still interactable